Notes_and_Examples/1_protractor_singleVector/1_protractor.xml

```diff
@@ -42,8 +35,15 @@ def vglcfn(e, ans):
           return 'The length of {} is incorrect. Your length: {:.1f}'.format(vec.name, vec.length)
   def check_angle(check, vectors):
       vec = vectors[check['vector']]
+      expected = check['expected']
       tolerance = check.get('tolerance', 2.0)
-      if abs(vec.angle - check['expected']) > tolerance:
+      upper_limit = (expected + tolerance) % 360
+      lower_limit = (expected - tolerance) % 360
+      if upper_limit < lower_limit:  # Can happen when upper_limit=2, lower_limit=-358, for example.
+          valid = vec.angle <= upper_limit or vec.angle >= lower_limit
+      else:
+          valid = vec.angle >= lower_limit and vec.angle <= upper_limit
```

Comment:
@mtyaka nit: not sure, but likely can be written as lower_lmit <= vec.angle <= upper_limit - python allows that so if JS input problem does not interfere it should juist work